tests/test_main.cpp

```diff
@@ -124,6 +123,7 @@
 #include "tests/test_validate_testing.h"
 
 #ifndef _3D_DISABLED
+#include "tests/scene/test_arraymesh.h"
```

Comment:
The tests for `ArrayMesh` all depends on 3D specific code, like the primitive meshes, so there's no use testing it at all without 3D enabled